setup.py

```diff
@@ -28,9 +28,7 @@ def run_tests(self):
     'six>=1.5',
     'contextlib2; python_version=="2.7"',
     'mock; python_version=="2.7"',
-    'yarl; python_version>"3.4"',
-    'yarl<1.0.0; python_version=="3.4"',
-    'multidict<4.0.0,>=2.0; python_version=="3.4"'
+    'yarl; python_version>"3.5"',
```

Comment:
Does this mean yarl is no longer installed for Python 3.5? Shouldn't this be a `>=` instead?

Reply:
Thank you!